master/cache.lua

```diff
@@ -114,7 +114,7 @@ local function fetch_and_store_marathon_apps()
        local appId = app["id"]
        local labels = app["labels"]
        if not labels then
-          ngx.log(ngx.NOTICE, "Labels not found in app '" .. appId .. "': " .. err)
+          ngx.log(ngx.NOTICE, "Labels not found in app '" .. appId .. "'")
```

Comment:
Nice, this whole code section was originall created in one big bad copy pasta scenario, I believe to remember.